tests/runtime/json-output

```diff
@@ -190,3 +190,14 @@ NAME print bool type
 RUN {{BPFTRACE}} -q -f json -e 'BEGIN { print((true, false)); exit(); }'
 EXPECT {"type": "value", "data": [true,false]}
 TIMEOUT 1
+
+NAME BENCH
+RUN {{BPFTRACE}} -q -f json --test-mode bench -e 'BENCH:a { @ = count(); }'
```

Comment:
Maybe add a new test with a superfluous probe to verify that it did not attach and maybe add a `BEGIN` or `END` to make sure they do run and in the right order.